gelfclient/client.py

```diff
@@ -44,6 +50,9 @@ def log(self, _fields_dict = {}, **fields_named):
                 message['host'] = message['source']
             else:
                 message['host'] = self.source
+        if 'level' not in message:
+            message['level'] = LogLevel.alert
```

Comment:
Not setting any default value does indeed sound better - more in tune with how Greylog works. 

I have not seen your request to remove the LogLevel class, where is that? 
The goal of it was to make logs level explicit for the clients of this library, so they don't have to remember that 3 means error or wonder what levels are allowed etc. With syslog I guess the only thing is to make sure the levels are the same with Graylog's (at this point I don't know).